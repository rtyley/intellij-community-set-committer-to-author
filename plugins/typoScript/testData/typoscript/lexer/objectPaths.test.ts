myObject.prop1.prop2.prop3.prop4.prop5 < otherObject.prop
.10 > not.a.path
9path.%BAD_SYMBOL%.prop < not a path.because.of.spaces
