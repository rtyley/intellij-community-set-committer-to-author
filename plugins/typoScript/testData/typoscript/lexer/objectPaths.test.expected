object path entity ('myObject')
. ('.')
object path entity ('prop1')
. ('.')
object path entity ('prop2')
. ('.')
object path entity ('prop3')
. ('.')
object path entity ('prop4')
. ('.')
object path entity ('prop5')
WHITE_SPACE (' ')
< ('<')
WHITE_SPACE (' ')
object path entity ('otherObject')
. ('.')
object path entity ('prop')
WHITE_SPACE_WITH_NEW_LINE ('
')
. ('.')
object path entity ('10')
WHITE_SPACE (' ')
> ('>')
IGNORED_TEXT (' not.a.path')
WHITE_SPACE_WITH_NEW_LINE ('
')
object path entity ('9path')
. ('.')
BAD_CHARACTER ('%')
object path entity ('BAD_SYMBOL')
BAD_CHARACTER ('%')
. ('.')
object path entity ('prop')
WHITE_SPACE (' ')
< ('<')
WHITE_SPACE (' ')
object path entity ('not')
WHITE_SPACE (' ')
object path entity ('a')
WHITE_SPACE (' ')
object path entity ('path')
. ('.')
object path entity ('because')
. ('.')
object path entity ('of')
. ('.')
object path entity ('spaces')
WHITE_SPACE_WITH_NEW_LINE ('
')